### PR TITLE
Add form validations

### PR DESCRIPTION
**Pull request by Abdo9826**
Add Form.js file

### CODE REVIEW

File: form.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const containerform = document.querySelector('.input');
+const error = document.querySelector('#error');
+containerform.addEventListener('submit', (e) => {
```
Comment (Ismailco) [inline form.js:3]: - [ ] If a user submits the contact form with the email field containing uppercase characters an error message should show up, please implement this functionality it's a requirement

File: form.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const containerform = document.querySelector('.input');
```
Comment (Ismailco) [inline form.js:1]: - [ ] If a user submits the contact form with the email field containing only lowercase characters the form is sent normally. for now, the form doesn't submit at all, kindly fix this so the form is sent when the user clicks the submit button

File: form.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        && emailArray[i] !== '@' && emailArray[i] !== '.'
+    ) {
+      right = false;
+    }
+  }
+  if (right) {
+    error.innerHTML = '';
+  } else {
+    e.preventDefault();
+    error.innerHTML = 'Please enter your email in lower case only';
```
Comment (Ismailco) [inline form.js:19]: - [ ] This error message should show up close to the "submit" button, as for now it does not, please fix this so the error message displayed on the form when there is capital letters on the email

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Abdo9826,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Modifying the README and you can remove the empty headers so it is more professional

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KDlamini — APPROVED** [review]
### Hi @Abdo9826 🙋‍♀️,  

Great job! 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**